File: project2/Wolf.java
class Wolf extends Animal {
    public Wolf(int x, int y, World w) {
        super(9, 5, x, y, w);
    }
    
    @Override
    public char draw() {
        return 'W';
    }
    
    @Override
    public Animal clone(int x, int y) {
        return new Wolf(x, y, world);
    }
}

File: project3/wolf.py
from animal import Animal

class Wolf(Animal):
    def __init__(self, x, y, world):
        super().__init__(9, 5, x, y, world)
    
    def draw(self):
        return 'W'
    
    def clone(self, x, y):
        return Wolf(x, y, self.world)
